tests/components/switch/test_template.py

```diff
@@ -131,15 +132,11 @@ def test_template_syntax_error(self):
                 }
             }
         })
-
-        state = self.hass.states.set('switch.test_state', STATE_ON)
-        self.hass.pool.block_till_done()
```

Comment:
The old approach didn't validate the template until run time. The voluptuous does it at load time - so there isn't an entity to set the state of!